pkg/occlient/volumes_test.go

```diff
@@ -411,3 +369,190 @@ func TestRemoveVolumeMountsFromDC(t *testing.T) {
 		})
 	}
 }
+
+func TestGetPVCNameFromVolumeMountName(t *testing.T) {
+	type args struct {
+		volumeMountName string
+		dc              *appsv1.DeploymentConfig
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "Test case : Deployment config with given PVC",
+			args: args{
+				volumeMountName: "test-pvc",
+				dc: &appsv1.DeploymentConfig{
```

Comment:
Is it possible to use `fakeDeploymentConfig` here instead of specifying entire spec?

Reply:
Fixed